src/main/java/teammates/common/util/Templates.java

```diff
@@ -504,7 +504,10 @@ public static String populateTemplate(String template, String... keyValuePairs)
     // TODO: Consider adding instructions for the feedback session into template?
     // TODO: Or simply use static strings here?
     public static class FeedbackSessionTemplates {
-        public static final String TEAM_EVALUATION =
+        public static final String TEAM_EVALUATION_TEMPLATE =
```

Comment:
Can drop the `_TEMPLATE` suffix as this is already in a class named `FeedbackSessionTemplates`.